src/components/home-header.tsx

```diff
@@ -0,0 +1,40 @@
+import React, {useContext} from 'react'
+import {View, Text} from 'react-native'
+import {containerStyles, navigation} from '../styles'
+import {UserContext} from '../providers/user.provider'
+import {FormattedMessage} from 'react-intl'
+
+export const HomeHeaderTitle = () => {
+  const {user} = useContext(UserContext)
+
+  const hasFullName = user?.full_name ? true : false
+  const hasPasswordDigest = user?.password_digest ? true : false
+
+  return (
+    <View
+      style={[
+        {
+          ...containerStyles.fill,
+          justifyContent: 'center',
+          alignItems: 'center',
+        },
+      ]}>
```

Comment:
```[
        containerStyles.fill,
        {
          justifyContent: 'center',
          alignItems: 'center',
        },
      ]```

Is this the same?

Reply:
Nope

Reply:
This doesn't have anything to do with flexbox?

If you use the inline style array notation it combines the objects, so no need to spread them.

As shown here: 

<img width="711" alt="Screenshot 2020-04-08 at 13 14 13" src="https://user-images.githubusercontent.com/9961990/78783008-dabd8780-799a-11ea-9ed9-5b0bbb4c9e51.png">